package.json

```diff
@@ -39,6 +41,7 @@
   },
   "lint-staged": {
     "**/*": [
+      "suppress-exit-code markdownlint --fix",
```

Comment:
Using [suppress-exit-code](https://www.npmjs.com/package/suppress-exit-code) here so that it was possible to commit changes even if there are markdownlint issues. We can remove this later when we are sure that markdowlint is a must-have tool in this repo.